src/AccessibilityInsights.SharedUx/Dialogs/HotkeyGrabDialog.xaml

```diff
@@ -28,19 +27,20 @@
             <RowDefinition Height="Auto"/>
         </Grid.RowDefinitions>
         <Label Content="{x:Static Properties:Resources.HotkeyGrabDialogWindowTitle}" HorizontalAlignment="Left" Margin="8,8,0,0" Grid.Row="0" VerticalAlignment="Top" Style="{StaticResource LblH5}" Padding="0"/>
-        <TextBlock HorizontalAlignment="Left" Margin="8,10,0,0" TextWrapping="Wrap" 
+        <TextBox Grid.Row="2" Margin="0,10" Name="tbHotkey"  
+                 Height="24" Width="300" HorizontalContentAlignment="Center"
+                 VerticalContentAlignment="Center" Style="{StaticResource StandardTextBox}"
+                 AutomationProperties.Name="{x:Static Properties:Resources.HotkeyGrabDialogWindowTitle}"
+                 AutomationProperties.HelpText="{x:Static Properties:Resources.RunTextToChangeKeyBoard}"
+                 KeyDown="Window_KeyDown" KeyUp="Window_KeyUp" 
```

Comment:
Should the names of these handlers be updated now that they aren't just Window's handlers?